### ISSUE TITLE
Modern Warfare Scope Incompatibility

### ISSUE THREAD

**Issue by MalTeeez** [Abastro/StellarSky#119]
The Modern Warfare scopes are moved the right so shots misalign with the actual path

**Comment by McParrot69** [comment]
Same problem i replace with this mod https://www.curseforge.com/minecraft/mc-mods/gog-skybox.
U solve the problem?

**Comment by MalTeeez** [comment]
No, not possible since both interact with the GL Code in the same context, and MW is not really handling any other interaction with that well.
